terms/hyperparameters_nm.tex

```diff
@@ -0,0 +1,70 @@
+\chapter[Hyperparameters]{Hyperparameters}
+\label{ch:Hyperparameters}
+
+As stated by Li \textit{et al.} in \cite{Li2017}, "performance of machine learning algorithms depends critically on identifying a good set of hyperparameters". Hyperparameter tuning is a topic often discussed in literature describing any combination of machine learning algorithms. Adequate values set for hyperparameters will massively influence training of a model, be it in terms of processing time, as well as the actual robustness of the model in question. 
+
+\section{Introduction}\label{sec:page-layout}
+\subsection{Defining Hyperparameters}\label{sec:headings}\index{headings}
+Hyperparameters are variables which are initialised before a machine learning model undergoes training. The general representation is as a vector, with common examples of hyperparameters including the learning rate of a model, regularization coefficients, and kernel parameters. Considering the learning rate, an improperly set value will greatly influence the training potential of a regression model or neural network. Extremely high learning rate values may render a system volatile, causing weight/parameter values to fluctuate very easily, whilst lower values may inhibit learning capabilities. Similarly, altering the regularization coefficient for a Support Vector Machine (SVM) may greatly affect the shape of its decision boundary.
+
+
+\section{Approaches for Hyperparameter Optimization}
+Owing to the necessity of optimal hyperparameter values in machine learning problems, various works of literature are dedicated solely to their optimization. During the past decade, methods have been refined and applied in a large number of model methods. The sections below will go through a number of these approaches found in published works.
```

Comment:
> Owing to the necessity of optimal hyperparameter values in machine learning problems, various works of literature are dedicated solely to their optimization.

You cannot put in something like this and not put in a few references to strengthen your point.